packages/modules/src/modules/banners/common/BannerWrapper.tsx

```diff
@@ -84,6 +98,26 @@ const withBannerData = (
         }
     }, [submitComponentEvent]);
 
+    const cleanParagraphsOrMessageText = (
+        paras: string[] | undefined,
+        text: string | undefined,
+    ): string[] => {
+        const originalCopy = getParagraphsOrMessageText(paras, text);
+
+        if (!originalCopy.length) {
```

Comment:
this `if` block is unnecessary - `originalCopy.map()` will return the array unchanged if it's empty

Reply:
Fixed as suggested